# Do not equate warnings to errors when running the tests in Debian
--- a/pytest.ini
+++ b/pytest.ini
@@ -43,8 +43,6 @@
 doctest_optionflags = ALLOW_UNICODE ELLIPSIS
 
 filterwarnings =
-  error
-
   # pytest>=6.2.0 under Python 3.8:
   # Ref: https://docs.pytest.org/en/stable/usage.html#unraisable
   # Ref: https://github.com/pytest-dev/pytest/issues/5299
